distributed-system-project/http-blob-server/blob/handlers/post.handler.ts

```diff
@@ -64,7 +66,22 @@ export async function postBlobHandler(
       errorMessage: `Invalid ID format. Only alphanumeric characters, periods, underscores, and hyphens are allowed, with a maximum length of ${config.MAX_ID_LENGTH} characters.`,
     });
   }
+  try {
 
-  await BlobService.createBlob(request.params.id, request, headers);
-  return reply.code(204).send();
+    logger.debug({
+      blobId: request?.params?.id,
+      msg: 'saving blob',
+    });
+    await BlobService.createBlob(request.params.id, request, headers);
```

Comment:
coding/design: `BlobService.createBlob` should not be aware of Fastify. consider passing only `request.raw`